### DISCUSSION TITLE
How can I remove footer, main menu, help icon and lock the canvas if viewModeEnabled is true?

### DISCUSSION THREAD

**Discussion by sankarbiswas07** [excalidraw/excalidraw#8809] [General]
Hey, I am exploring Excalidraw, it's an awesome tool. 
I was wondering If I could remove/ hide- the footer, main menu, and help icon and lock the canvas if viewModeEnabled is true? 
Is this possible though? If yes, can you guide me?

**Comment by Mrazator** [comment]
Hi, I think you would have to fork for these kind of modifications.

**Reply by sankarbiswas07** [reply]
OK. Thanks for the direction.
